webapp/src/translations/languages/en.json

```diff
@@ -658,7 +658,7 @@
         "back": "Back",
         "verifyMnemonicSeed": "Verify mnemonic seed",
         "verifyMnemonicGuideline": "To ensure you have written down your 24-word mnemonic seed, please identify which 6 words are in your mnemonic seed.",
-        "verifyMnemonicWhichOne": "Which one is…",
+        "verifyMnemonicWhichOne": "Which one is ...",
```

Comment:
"Which one is..." should remain as "Which one is...", no `space`. However, the `.` used is correct.

Reply:
You are right. I did some research and removed the space before the ellipsis.